src/Symfony/Component/Validator/Constraints/EmailValidator.php

```diff
@@ -118,7 +118,7 @@ public function validate($value, Constraint $constraint)
      */
     private function checkMX($host)
     {
-        return checkdnsrr($host, 'MX');
+        return !empty($host) && checkdnsrr($host, 'MX');
```

Comment:
I know but `substr($value, strrpos($value, '@') + 1)` with `$value = foo@bar.fr@` returns `""` with PHP 7.* and `false` with PHP 5.6. So only checking for string emptiness is not enough.